lib/netext/httpext/batch.go

```diff
@@ -57,7 +57,11 @@ func MakeBatchRequests(
 	result := make(chan error, reqCount)
 	perHostLimiter := lib.NewMultiSlotLimiter(perHostLimit)
 
-	makeRequest := func(req BatchParsedHTTPRequest) {
+	makeRequest := func(req *BatchParsedHTTPRequest) {
+		if req == nil {
+			result <- nil
+			return
```

Comment:
We might want to optimize this so that a goroutine isn't created at all for nil `req`s, but that would involve some shuffling of values around to ensure they match the index of the request as specified in the script, so it was easier to just exit early.

Reply:
No longer an issue after f2665be0.